rrmngmnt/ssh.py

```diff
@@ -112,6 +112,17 @@ def _update_timeout_exception(self, ex, timeout=None):
             ex.args = (message,)
             ex._updated = True
 
+        def get_transport(self, timeout=None):
```

Comment:
Where are you using this method ?
Or for what you want to use it ?
Please add doc string.